capstone/backend/src/main/java/com/google/servlets/ShareViaEmailServlet.java

```diff
@@ -0,0 +1,116 @@
+package com.google.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/v1/share-via-mailjet")
+public class ShareViaEmailServlet extends HttpServlet {
+
+  private String emailRegEx = "^[^@]+@[^@]+.[^@]+$";
+
+  private class RequestBody {
+    private String email;
+    private String html;
```

Comment:
This is ok for a V1, to wire up the email servlet, though the request should probably take in the list of items (stores with items you want to send), and you can construct the html string (from a html template file) on the backend.

For example:
https://stackoverflow.com/questions/5936003/write-html-file-using-java/15368262#:~:text=For%20Example%3A,close()%3B

Reply:
Thank you for this, I was planning on making an HTML string that just pops in the store information so this is super helpful